tokenserver/assignment/sqlnode/sql.py

```diff
@@ -634,35 +625,34 @@ def get_best_node(self, service, email):
         """Returns the 'least loaded' node currently available, increments the
         active count on that node, and decrements the slots currently available
         """
-        if self.allocate_to_spanner(email):
-            return (
-                self.settings.get("spanner_node_id", 800),
-                self.settings.get("spanner_entry", None)
-            )
-
         nodes = self._get_nodes_table(service)
         service = self._get_service_id(service)
 
-        # Pick the least-loaded node that has available slots.
-        where = [nodes.c.service == service,
-                 nodes.c.available > 0,
-                 nodes.c.capacity > nodes.c.current_load,
-                 nodes.c.downed == 0,
-                 nodes.c.backoff == 0]
+        query = select([nodes])
 
-        query = select([nodes]).where(and_(*where))
-
-        if self._is_sqlite:
-            # sqlite doesn't have the 'log' funtion, and requires
-            # coercion to a float for the sorting to work.
-            query = query.order_by(nodes.c.current_load * 1.0 /
-                                   nodes.c.capacity)
+        if self.allocate_to_spanner(email):
+            # Some users get allocated directly to the new spanner node.
+            query = query.where(nodes.c.id == self._spanner_node_id)
```

Comment:
This looks up the metadata for the spanner node in the db, rather than having to duplicate it all into the config file. We were already hitting the db here in a pre-spanner world to find the best node, so I figure there won't be any perf impact to doing this lookup on every request. We could optimize to only do it once if desired.